src/game_character.cpp

```diff
@@ -749,10 +745,6 @@ void Game_Character::BeginJump(int32_t& current_index, const RPG::MoveRoute& cur
 	SetStopCount(0);
 	SetMaxStopCountForStep();
 	move_failed = false;
-
-	if (IsContinuous()) {
-		SetAnimFrame(RPG::EventPage::Frame_middle);
-	}
```

Comment:
Before a jump starts the animation is reset to the Middle frame.
Did you move this fix somewhere else?

Reply:
This was moved to `UpdateAnimation()` under `if (IsJumping())`.

The logic here is wrong. It always resets the `anim_frame` (and `anim_count` too), regardless of your movement type.

Reply:
Also I verified if you're movement type is `fixed_graphic`, jump doesn't reset the `anim_frame` or `anim_count`. That's handled `UpdateAnimation()`